docs/source/key_components/dashboard_visualization.rst

```diff
@@ -71,27 +73,22 @@ To start this visualization tool, user need to input command following the forma
 
 .. code-block:: sh
 
-    maro inspector dashboard --source_path {source\_folder\_path} --force {true/false}
-
-----
-
-e.g.
-
-.. code-block:: sh
-
-    maro inspector dashboard --source_path .\maro\dumper_files --force false
+    maro inspector dashboard --source_path SNAPSHOT_DUMP_DATA_FOLDER --force {true/false}
 
 ----
 
 Parameter **force** refers to regenerate cross-epoch summary data or not, default value is 'true'.
-Parameter **source_path** refers to the path of dumped snapshot files.
-The expected structure of file folder should be like this:
+Parameter **source_path** refers to the path of dumped snapshot files. Every experiment would
+generate a data folder named by a timestamp. The input of the parameter **source_path** should be
```

Comment:
How about change to

"Every experiment would generate a data folder with a timestamp as the suffix, which we refer as SNAPSHOT_DUMP_DATA_FOLDER."

?